docs/usage/sat_bootprep.md

```diff
@@ -556,14 +556,14 @@ follows.
 
 The default bootprep input file for management NCNs (`management-bootprep.yaml`)
 uses the value `management-{{recipe.version}}` as the name of the CFS
-configuration. This uses a Jinja2 template to include the HPC CSM Software
+configuration. This value uses a Jinja2 template to include the HPC CSM Software
 Recipe version in the name of the CFS configuration. For example, when `sat
-bootprep` is run against this file in HPC CSM Software Recipe version `23.03`, a
-configuration named `management-23.03` is created.
+bootprep` is run against the default bootprep input file in HPC CSM Software
+Recipe version `23.05`, a configuration named `management-23.05` is created.
 
 This default management CFS configuration name might be acceptable for your
-system. However, it is possible to use a different names. `sat bootprep` creates
-whatever configurations are specified in the input file. For example, to create
+system. However, it is possible to use a different name. `sat bootprep` creates
+whatever configuration is specified in the input file. For example, to create
```

Comment:
IMO, it is still more correct to refer to `configurations` as a plural. bootprep input files contain a list of configurations and creates whatever is in the list. That being said, for management configuration I think the default is a list of one, so it's sort of correct either way.

Reply:
It might be more precise to specify "any and all configurations in the input file" here instead, as there isn't (yet) a way to limit or skip arbitrary configurations without editing the input file.

Reply:
Thanks for the suggestion, Jack! I'll update this in the next commit.